src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -55,6 +57,19 @@ public Tag(String tagName) {
     public static boolean isValidTagName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
+    /**
+     * updates the hashmap containing the alias and fulltagname
```

Comment:
Invalid javadoc format
should be `Updates the hashmap containing the alias and FullTagName.`